balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/Cassandra11QueryImpl.scala

```diff
@@ -44,12 +42,15 @@ class Cassandra11QueryImpl(context: AstyanaxContext[Keyspace]) extends Cassandra
   def getAllEntityIds(recordType: RecordType, period: Period): Iterator[String] = {
     fastfail.proceedOrThrow()
     try {
-      val retVal: Iterator[String] = context.getEntity.prepareQuery(Cassandra11Util.getColumnFamily(period, recordType))
-        .setConsistencyLevel(ConsistencyLevel.CL_ONE)
-        .withRetryPolicy(new ExponentialBackoff(250, 5)) // initial, max tries
```

Comment:
It looks like the retry policy isn't part of the query execution any more, however it seems to be set when establishing the cluster. The default seems to be the best choice, so that should be fine. However it would be nice to wrap that in the logging retry policy. If I've understood correctly, that should just add some information to the logs about what is happening.